SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -571,6 +585,28 @@ void USpatialWorkerConnection::ProcessOutgoingMessages()
 				nullptr);
 			break;
 		}
+		case EOutgoingMessageType::AddComponent:
+		{
+			FAddComponent* Message = static_cast<FAddComponent*>(OutgoingMessage.Get());
+
+			static const Worker_UpdateParameters DisableLoopback{ true /* loopback */ };
```

Comment:
```suggestion
			static const Worker_UpdateParameters EnableLoopback{ true /* loopback */ };
```

Reply:
Or something neutral like `UpdateParams`?

Reply:
I think EnableLoopback is good just to be explicit at what the params are intended for.